### ISSUE TITLE
v1.24.0 Always shows "Video load failed"

### ISSUE THREAD

**Issue by hawkeye64** [DIYgod/DPlayer#303]
Upgraded from v1.23.0. No code changes. Every time a video is played, the notice "Video load failed" appears. The video plays fine, but notice stays on screen.

What are the conditions this occurs and was there supposed to be any code changes I'm supposed to do from 1.23.0 to 1.24.0 that would prevent this error from displaying? I searched the docs and could not find anything. I also looked in the 1.24.0 release notes.

**Comment by DIYgod** [comment]
This notice appears when video event `error` triggered, can you provide a demo?

**Comment by hawkeye64** [comment]
Not really. We have remote cameras that are connected to a server, and then a client that uses a browser to access a UI. When the User wants to watch some of the video, we download from the camera to the server. I suspect, that the dplayer is passed in the url to where the video is downloading and it's getting this error because the video may not be ready at that time. It's a bit convoluted, therefore extremely difficult to write a jsfiddle that owuld do the same thing. At least this is my suspicion.
The thing is, the video does load up fine and can be played. I don't mind the warning either. I would just prefer it to go away once the player hits the ready state.

**Comment by hifocus** [comment]
You may see #307 , it might be some of your resources failed to load. Like cover images, in my case.

**Comment by hawkeye64** [comment]
I think it'd be nice to have an api call to reset this error. I don't mind if it shows up again, but I don't want it to stay there permanently. I have two players on the screen at the same time and they play syncronized video. The User can select which ones to play so it's dynamic. But, once the error appears, it seems it never go away.

**Comment by hawkeye64** [comment]
It looks like it was a result of this code:
```
 this.tran && this.notice && this.type !== 'webtorrent' & this.notice(this.tran('Video load failed'), -1); 
```
Which has incorrect `AND` notation (single &).

When will there be a release with this PR (https://github.com/MoePlayer/DPlayer/issues/307)? The build is still 1.24.0. :(

**Comment by 0obuwawao0** [comment]
v1.27.0
API
switchVideo({
    url : ''
})

After switching to the unplayable video url, and then switching to the normal playable video url, this bug still exists

**Comment by DIYgod** [comment]
Fixed in 255bbac7a4e0b423e91eb6c07aca146523bf61a6
